dashboard/src/components/AppView/DeploymentTable.tsx

```diff
@@ -4,29 +4,33 @@ import { IResource } from "../../shared/types";
 import DeploymentItem from "./DeploymentItem";
 
 interface IDeploymentTableProps {
-  deployments: { [d: string]: IResource };
+  deployments: IResource[];
```

Comment:
Could this be Ideployment[]?

Reply:
with deployments happen the same than services, since it follows the same layout it's not necessary to define a new interface.

Reply:
Sorry, I think that I do not understand this comment.